tasks/config/webpack.config.js

```diff
@@ -20,8 +20,7 @@ module.exports = {
     'react': 'React',
     'react-dom': true,
     'react-highlight-click': true,
-    'react-outsideclick': true,
-    'react-addons-css-transition-group': true
+    'react-outsideclick': true
```

Comment:
I don't think we can remove this. See Slideshow.js: 
`const Transition = require('react-addons-css-transition-group');`